tests/rt.conf

```diff
@@ -1,275 +1,198 @@
+#RUN     | cpld_control_c384                                                                                                      | standard | cheyenne.intel                          | fv3 |
+#RUN     | cpld_controlfrac_c384                                                                                                  | standard | cheyenne.intel                          | fv3 |
+#RUN     | cpld_bmark                                                                                                             | standard | cheyenne.intel                          | fv3 |
+
+COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y WW3=Y                             | standard | + hera.intel orion.intel wcoss_dell_p3  | fv3 |
+#COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y WW3=Y                            | standard | cheyenne.intel                          | fv3 |
+RUN     | cpld_bmark_wave                                                                                                         | standard | + hera.intel orion.intel wcoss_dell_p3  | fv3 |
+#RUN     | cpld_bmark_wave                                                                                                        | standard | cheyenne.intel                          | fv3 |
+
+COMPILE | DEBUG=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y                           | standard | + hera.intel orion.intel wcoss_dell_p3  | fv3 |
+#COMPILE | DEBUG=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y                          | standard | cheyenne.intel                          | fv3 |
+RUN     | cpld_debug                                                                                                              | standard | + hera.intel orion.intel wcoss_dell_p3  | fv3 |
+#RUN     | cpld_debug                                                                                                             | standard | cheyenne.intel                          | fv3 |
+
+##############################################################################################################################################################################################
+# Data Atmosphere tests                                                                                                                                                                      #
+##############################################################################################################################################################################################
+
+COMPILE | DATM=Y S2S=Y                                                                                                            | standard | + hera.intel orion.intel wcoss_dell_p3  | fv3 |
```

Comment:
I am curious, do we still allow platforms specified without "+" or "-", I feel the "+ platforms" is the same as "platforms", is it correct?

Reply:
As far as I understand the code changes in `rt.sh`, a line can either be entirely empty (in which case it means all machines) or it **must** have a `+` or a `-` preceding a list of machine names. If there is no `+` or `-` and the line is not empty, this will create an error.